### DISCUSSION TITLE
How to implement responsive list grid in ftd for different screens?

### DISCUSSION THREAD

**Discussion by Sarvom** [fastn-stack/fastn#848] [Q&A] [answered]
How to implement responsive list grid in ftd for different screens? (3x2,2x3,1x6 etc)

**Comment by amitu** [comment, marked as answer]
You can use the [`wrap` ](https://fastn.io/container-attributes/#wrap-optional-boolean) to make a row wrap. Then you will have to ensure the width of each of elements and padding on row is such that only desired number of elements fit per row.
